volunteers/templates/static/promo.html

```diff
@@ -3,38 +3,38 @@
 <div id="intro">
     <h2 class="intro"><span>FOSDEM</span><br/>Volunteering</h2>
 
-    <p class="intro">This is a site that brings all the volunteers together.
-    Find all the cool tasks that you are interested in.</p>
+    <p class="intro">This site brings all volunteers together.
+    Find cool tasks you are interested in.</p>
 
-    <p id="xtra-info">Problems? For bugs you can report an issue at
-    <a href="https://github.com/FOSDEM/volunteers/issues">Github</a>.</p>
+    <p id="xtra-info">Problems? Fix or report bugs as issues at
+    <a href="https://github.com/FOSDEM/volunteers/issues">GitHub</a>.</p>
 </div><!-- /intro-->
 
 <ul class="features">
     <li><h3>FOSDEM</h3>
-        <p><a href="https://www.fosdem.org">FOSDEM</a> is a free event that offers open source
+        <p><a href="https://www.fosdem.org">FOSDEM</a> is a gratis event that offers libre software
         communities a place to meet, share ideas and collaborate.</p>
     </li>
 
-    <li><h3>Location</h3>
-        <p>FOSDEM will take place at the<a href="https://fosdem.org/practical/transportation/">
+    <li><h3><a href="https://www.openstreetmap.org/way/30358555#map=15/50.8207/4.3966">Location</a></h3>
```

Comment:
I'm afraid we should defer to the main website for venue information so I'd drop this line.